### ISSUE TITLE
Section 508 Accessibility Issue - Loss of Focus for Menu Items

### ISSUE THREAD

**Issue by aglynn** [gwtproject/gwt#9252]
I am using Google Web Toolkit 2.5.1 I believe. (You can see it at the top of https://github.com/AppVet/appvet/blob/master/src/gov/nist/appvet/gwt/appvet.gwt.xml)

This ticket relates to accessibility. There is an issue that when I use GWT on my site I cannot tell where I am when I am tabbing through the page. Specifically, the visual indicator of the focus (the item you are on when you are tabbing) disappears when I tab onto the menu. 

I highlighted "Anthony" in the attached screenshot because at that point the focus has disappeared. 

![lostfocus](https://cloud.githubusercontent.com/assets/1056178/11966747/9c7fc748-a8cb-11e5-91e3-7af7f62cf91e.PNG)

The only way I know the focus is on that item is because when I press Enter on my keyboard the menu opens up. (The following screenshot shows that.)

![lostfocus2](https://cloud.githubusercontent.com/assets/1056178/11966746/9c7b05dc-a8cb-11e5-9bba-b89e78898e6b.PNG)

You can see the code at the following link by looking at the "appVetMenuBar": https://github.com/AppVet/appvet/blob/master/src/gov/nist/appvet/gwt/client/gui/AppVetPanel.java#L897

This related to Section 508 in this way: 1.1 Keyboard Access	1194.21 (Software Applications and Operating Systems) a (Executing Function From Keyboard), c (Input Focus), 1194.31 (Functional performance) a (Blindness), b (Low vision) Baselines: B01, B02, B03, B04. Specifically, it related to E. **At any time, there is no visual indication of the current focus (loss of focus) (21(c), B02)

**Comment by aglynn** [comment]
I just wanted to follow up to see if this issue could be triaged? I know you guys have a lot on your plate so I appreciate any assistance. Thank you.

**Comment by jonl-percsolutions-com** [comment]
This should be more to due with the css applied by the "Clean" theme than a "bug" of any kind.  You will either need to override the clean theme styles or extend/create your own theme.  Check out http://www.gwtproject.org/doc/latest/tutorial/style.html

**Comment by AppVet** [comment]
I was able to resolve this by using GWT Roles. For each MenuItem, I simply set the tabIndex to 0. For example:

```
final MenuBar myMenuBar = new MenuBar();
final MenuItem helpMenuItem= new MenuItem("Help", true, myMenuBar);
Roles.getHeadingRole().setTabindexExtraAttribute(helpMenuItem.getElement(), 0);
```
